### PR TITLE
feat: tile cooldown support

### PR DESCRIPTION
**Pull request by melancholiai**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✖                                                                        |
| New feature     | ✔                                                                        |
| Breaking change | ✖                                                                        |
| Deprecations    | ✖                                                                        |
| Documentation   | ✔                                                                        |
| Tests added     | ✔                                                                        |
| Chore            | ✖                                                                       |

### CODE REVIEW

File: packages/common/src/types/index.ts [hunk 1]
```diff
@@ -1,8 +1,12 @@
+import { Geometry } from 'geojson';
+
 interface LogFn {
   (obj: unknown, msg?: string, ...args: unknown[]): void;
   (msg: string, ...args: unknown[]): void;
 }
 
+type BBox = [number, number, number, number];
```
Comment (NivGreenstein) [inline packages/common/src/types/index.ts:8]: Use `BoundingBox` type from `geojson` package instead

File: packages/backend/src/common/util.ts [hunk 2]
```diff
@@ -47,3 +48,9 @@ export const stringifyCoordinates = (coordinates: LonLat): string =>
   `${coordinates.lon.toFixed(COORDINATES_FRACTION_DIGITS)}, ${coordinates.lat.toFixed(COORDINATES_FRACTION_DIGITS)}`;
 
 export const keyfy = (params: TileParamsWithKit): string => `${TILE_DETAILS_KEY_PREFIX}:${params.kit}:${params.z}/${params.x}/${params.y}`;
+
+export const hashValue = (value: unknown): string => {
+  const hash = createHash('md5');
```
Comment (NivGreenstein) [inline packages/backend/src/common/util.ts]: MD5 hash as been proven to be vurnable to a collision attack. We might have to look for another hash function.

**Review by NivGreenstein — APPROVED** [review]

File: packages/backend/src/common/util.ts (hunk 2 above)
Comment (melancholiai) [reply packages/backend/src/common/util.ts]: done.

File: packages/common/src/types/index.ts (hunk 1 above)
Comment (melancholiai) [reply packages/common/src/types/index.ts:8]: geojson's BBox is either `[number, number, number, number] | [number, number, number, number, number, number]`, I prefer to have more precision

**Pull request merged**